requirements/requirements.txt

```diff
@@ -1,7 +1,7 @@
 boto3==1.*
 holidays==0.9.*
 matplotlib==3.*
-mxnet>=1.3.1,<1.5.*
+mxnet>=1.4.1,<1.5.*
```

Comment:
Do we need to update the version somewhere else for the CI?

Reply:
I am not sure; let me check.